browser/app/js/components/Browse.js

```diff
@@ -497,338 +508,349 @@ export default class Browse extends React.Component {
-            </Modal>
-            <Modal className="modal-about modal-dark"
-              animation={ false }
-              show={ showAbout }
-              onHide={ this.hideAbout.bind(this) }>
-              <button className="close" onClick={ this.hideAbout.bind(this) }>
-                <span>×</span>
-              </button>
-              <div className="ma-inner">
-                <div className="mai-item hidden-xs">
-                  <a href="https://minio.io" target="_blank"><img className="maii-logo" src={ logo } alt="" /></a>
-                </div>
-                <div className="mai-item">
-                  <ul className="maii-list">
-                    <li>
-                      <div>
-                        Version
-                      </div>
-                      <small>{ version }</small>
-                    </li>
-                    <li>
-                      <div>
-                        Memory
-                      </div>
-                      <small>{ memory }</small>
-                    </li>
-                    <li>
-                      <div>
-                        Platform
-                      </div>
-                      <small>{ platform }</small>
-                    </li>
-                    <li>
-                      <div>
-                        Runtime
-                      </div>
-                      <small>{ runtime }</small>
-                    </li>
-                  </ul>
-                </div>
+            <div className="preview__body">
+              <div className="preview__item preview__item--img">
+                <img src="https://placeholdit.imgix.net/~text?w=100&h=300" alt="" />
               </div>
-            </Modal>
-            <Modal className="modal-policy"
-              animation={ false }
-              show={ showBucketPolicy }
-              onHide={ this.hideBucketPolicy.bind(this) }>
-              <ModalHeader>
-                Bucket Policy (
-                { currentBucket })
-                <button className="close close-alt" onClick={ this.hideBucketPolicy.bind(this) }>
-                  <span>×</span>
+              <div className="preview__info">
+                <dl>
+                  <dt>Type</dt>
+                  <dd>
+                    JPEG Image
+                  </dd>
+                  <dt>Size</dt>
+                  <dd>
+                    1.1 MB
+                  </dd>
+                  <dt>Modified</dt>
+                  <dd>
+                    Mar 1, 2017 12:!2 PM
+                  </dd>
+                  <dt>Policy</dt>
+                  <dd>
+                    Read and Write
+                  </dd>
+                  <dt>Shared</dt>
+                  <dd>
+                    No
+                  </dd>
+                </dl>
+              </div>
+              <div className="preview__actions">
+                <button className="btn btn--lg btn--primary">
+                  Preview
```

Comment:
Ditto from [the last PR](https://github.com/minio/minio/pull/4697#discussion_r128390228), this may need different labeling?